djstripe/event_handlers.py

```diff
@@ -40,38 +46,51 @@ def customer_event_attach(event, event_data, event_type, event_subtype):
 
 @webhooks.handler("customer")
 def customer_webhook_handler(event, event_data, event_type, event_subtype):
-    """ Handles updates for customer objects. """
+    """Handle updates to customer objects.
 
+    First determines the crud_type and then handles the event if a customer exists locally.
+    As customers are tied to local users, djstripe will not create customers that
+    do not already exist locally.
+
+    Docs and eg. customer webhook response: https://stripe.com/docs/api#customer_object
+    """
     crud_type = CrudType.determine(event_subtype, exact=True)
     if crud_type.valid and event.customer:
-        # As customers are tied to local users, djstripe will not create
-        # customers that do not already exist locally.
         _handle_crud_type_event(target_cls=Customer, event_data=event_data, event_subtype=event_subtype, crud_type=crud_type)
 
 
 @webhooks.handler("customer.source")
-def customer_source_webhook_handler(
-        event, event_data, event_type, event_subtype):
-    """ Handles updates for customer source objects. """
+def customer_source_webhook_handler(event, event_data, event_type, event_subtype):
+    """Handle updates to customer payment-source objects.
 
+    Docs: https://stripe.com/docs/api#customer_object-sources.
+    """
     source_type = event_data["object"]["object"]
 
-    # TODO: other sources
+    # TODO: handle bitcoin_receiver_object (https://stripe.com/docs/api#bitcoin_receiver_object)
```

Comment:
interpreted `eg. Card or Bitcoin` as exclusive but ya there are totally more sources, will change this.